index.html

```diff
@@ -0,0 +1,122 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Formula 1 Get Together Conference</title>
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="hamburger-logo">
+    
+            <div class="header-container">
+                <img src="images/mobile_menu1.svg" class="ham">
+                <nav>
+                <img src="images/F1_logo.png" class="logo">
+                </nav>
+            </div>
+         
+
+        <nav class="menu" id="menu-pop">
+            <img src="images/Icon-close.svg" alt="" class="close-button">
+            <ul class="nav-content">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#program" id="nav-grey">Program</a></li>
+                <li><a href="#join" id="nav-grey">Join</a></li>
+                <li><a href="#sponsor" id="nav-grey">Sponsor</a></li>
+                <li><a href="#news" id="nav-grey">News</a></li>
+            </ul>
+        </nav>
+
+    </header>
+
+    <section class="intro">
+        <p id="hello">"Hello!,Sharing world"</p>
+        <div class="title-intro">
+            <h1>FORMULA 1 FANS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <div class="p-intro">
+            <p class="summit-intro">Welcome to the 7th global summit of Formula 1 fans.
+                Celebrate,have fun and bless someone.Venue:Vienna,Austria. </p>
+        </div>
+        <div class="date">
+            <p class="show-date">2023.9.8(FRI) ~ 9(SAT)</p>
+            <p>@Grand Prix of Vienna and more</p>
+        </div>
+    </section>
+
+    <section class="program">
+        <h2>Main program</h2>
+        <hr class="sm-line">
+
+        <div id="gr-program">
+        <div class="grid-program">
+            <img src="images/program_icon_01 1.svg" class="program-icon">
+            <p class="program-points">Lecture</p>
+            <p class="program-ex">Listen to the speakers from various countries 
+                about the messages of sharing and opening.</p>
+        </div>
+
+        <div class="grid-program">
+            <img src="images/program_icon_02 1.svg" class="program-icon">
+            <p class="program-points">CC Exhibition</p>
+            <p class="program-ex">Appreciate various creations applying 
+                CC license of artists, organized from Art Center Nabi</p>
+        </div>
+
+        <div class="grid-program">
+            <img src="images/program_icon_03 1.svg" class="program-icon">
+            <p class="program-points">Forum</p>
+            <p class="program-ex">Have the time to share your thoughts and 
+                opinions with experts for each topic.</p>
+        </div>
+
+        <div class="grid-program">
+            <img src="images/program_icon_04 1.svg" class="program-icon">
+            <p class="program-points">Workshop</p>
```

Comment:
[REQUIRED] Kindly ensure that your images contain alt attributes so that your design could be more accessible.

Reply:
Introduced alt attributes to the images